src/index.html

```diff
@@ -4,10 +4,59 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <title>Product cards</title><link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div data-qa="card" class="product-card">
+      <div class="product-card__image">
+        <img class="product-card__img"
+        src="./images/imac.jpeg"
+        alt="imac">
+      </div>
+
+      <h2 class="product-card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product-card__code">
+        Product code: 195434
+      </p>
+
+      <div class="product-stars">
+        <div class="product-stars__box stars stars--4">
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
```

Comment:
```suggestion
          <div class="stars__star"></div>
          <div class="stars__star"></div>
```

add blanks only between multiline elements